libs/sr/theme/src/Theme.tsx

```diff
@@ -0,0 +1,259 @@
+import { createTheme } from '@mui/material'
+
+import { theme as commonTheme } from '@genshin-optimizer/common/ui'
+
+declare module '@mui/material/styles' {
+  interface Palette {
+    brandSRO500: Palette['primary']
```

Comment:
brand colors and neutral colors should actually go into ui/common theme

Reply:
I noticed that this also exists in GO as a standalone theme library, hence why I generated one - should I also extract these values from that side as well then? I believe almost everything would be inherited from `common/ui` between the two except elemental dmg-related colors, and seemingly roll colors if we choose to make roll colors different for SRO specifically.

Reply:
I think its safe to think that we will share the same theme and colors assumptions in both SRO and GO, so i would agree that both can be moved to common/ui. If we have separate designs for SRO with separate theme, we'd extract it out again.